### PR TITLE
Line item field name changes

### PR DESCRIPTION
**Pull request by juanshishido**
The following are based on changes in [Ads API v9](https://twittercommunity.com/t/ads-api-version-9/150316):
* Add `goal`
* Remove `optimization` and `bid_unit`
* Rename:
    * `bid_type` -> `bid_strategy`
    * `charge_by` -> `pay_by`

### CODE REVIEW

File: twitter_ads/enum.py [hunk 1]
```diff
@@ -96,6 +76,23 @@ def enum(**enums):
...
+    APP_INSTALLS='APP_INSTALLS',
+    ENGAGEMENT='ENGAGEMENT',
+    FOLLOWERS='FOLLOWERS',
+    LINK_CLICKS='LINK_CLICKS',
+    MAX_REACH='MAX_REACH',
+    PREROLL='PREROLL',
+    PREROLL_STARTS='PREROLL_STARTS',
+    REACH_WITH_ENGAGEMENT='REACH_WITH_ENGAGEMENT',
+    VIDEO_VIEW='VIDEO_VIEW',
+    VIEW_3S_100PCT='VIEW_3S_100PCT',
```
Comment (tushdante) [inline twitter_ads/enum.py:90]: Didn't realize these are valid goals too (`VIEW_3S_100PCT`, `VIEW_6S` and `VIEW_15S`). These should be `pay_by` if I'm not mistaken

**Review by tushdante — APPROVED** [review]
LGTM! Just want to confirm the ENUM changes prior to merging

File: twitter_ads/enum.py (hunk 1 above)
Comment (juanshishido) [reply twitter_ads/enum.py:90]: Thanks for the callout. Let me double check on this.

**Comment by juanshishido** [comment]
@tushdante Should I update `API_VERSION` in [this file](https://github.com/twitterdev/twitter-python-ads-sdk/blob/master/twitter_ads/__init__.py) with this diff? Or should that happen separately?

File: twitter_ads/enum.py (hunk 1 above)
Comment (juanshishido) [reply twitter_ads/enum.py:90]: Confirmed that `VIEW_3S_100PCT` works when creating a line item. Also checked the field values internally. Thanks again for flagging!

**Comment by juanshishido** [comment]
> @tushdante Should I update `API_VERSION` in [this file](https://github.com/twitterdev/twitter-python-ads-sdk/blob/master/twitter_ads/__init__.py) with this diff? Or should that happen separately?

Will do this in a separate diff.

**Comment by ChanYiLin** [comment]
Hi, there is an issue for this change
While line item has removed 'bid_type' property, the `save` function in LineItem still checks the `bid_type` and causes property not found issue.

**Pull request merged**